ups/deploy_testing_distrib.table

```diff
@@ -0,0 +1,4 @@
+setupRequired(meas_deblender)
+setupRequired(obs_test)
+setupOptional(obs_cfht)
+setupOptional(obs_subru)
```

Comment:
We haven't been doing this for "metapackages" but technically `scons` and `sconsUtils` are deps.